tool/tctl/common/acl_command.go

```diff
@@ -70,6 +72,7 @@ func (c *ACLCommand) Initialize(app *kingpin.Application, _ *servicecfg.Config)
 	users := acl.Command("users", "Manage user membership to access lists.")
 
 	c.usersAdd = users.Command("add", "Add a user to an access list.")
+	c.usersAdd.Flag("refacl", "Add a reference to an access list to another access list").BoolVar(&c.isRefAcl)
```

Comment:
Instead of a boolean flag, I would use the same `--type=user|list` switch as on the backend.